darts/tests/models/forecasting/test_regression_models.py

```diff
@@ -403,15 +396,20 @@ def _apply_promo_mechanism(promo_mechanism):
             elif promo_mechanism == 10:
                 return np.random.normal(20, 3)
 
+        # fut_cov_promo_mechanism and target_qty are encoded as integers
         date_range = pd.date_range(start="2020-01-01", end="2023-01-01", freq="D")
         df = (
             pd.DataFrame({
                 "date": date_range,
                 "baseline": np.random.normal(100, 10, len(date_range)),
                 "fut_cov_promo_mechanism": np.random.randint(0, 11, len(date_range)),
                 "fut_cov_dummy": np.random.normal(10, 2, len(date_range)),
-                "past_cov_dummy": np.random.normal(10, 2, len(date_range)),
-                "past_cov_cat_dummy": np.random.normal(10, 2, len(date_range)),
+                "past_cov_dummy_neg_int": np.array([
```

Comment:
this neg int can probably removed once the validation check for it was removed, right?